target-platform/pom.xml

```diff
@@ -28,8 +28,8 @@
         <kura.addons.url>https://raw.github.com/eurotech/kura_addons/mvn-repo/</kura.addons.url>
         <skip.libsocket-can>true</skip.libsocket-can>
 
-        <maven.compiler.source>1.8</maven.compiler.source>
-        <maven.compiler.target>1.8</maven.compiler.target>
+        <maven.compiler.source>17</maven.compiler.source>
+        <maven.compiler.target>17</maven.compiler.target>
```

Comment:
We should remove Java 8 profile from this pom and set the java17 one as default settings of the plugin